torch_xla/csrc/device.cpp

```diff
@@ -55,6 +55,10 @@ std::string DeviceType::toString() const {
   return absl::StrCat(type_name_, ":");
 }
 
+XlaDeviceType DeviceType::getType() const {
```

Comment:
ditto

Reply:
Do we need a unit test for this function?